src/data/staking-products.json

```diff
@@ -268,6 +309,40 @@
         "eventAction": "Clicked",
         "eventName": "Clicked Launchnodes go to link"
       }
+    },
+    {
+      "name": "StakeWise",
+      "imageName": "Stakewise",
+      "hue": 220,
+      "launchDate": "2021-02-01",
+      "url": "https://docs.stakewise.io/for-operators/operator-service",
+      "audits": [],
+      "minEth": 0,
+      "isFoss": true,
+      "hasBugBounty": true,
+      "isTrustless": true,
+      "isPermissionless": true,
+      "multiClient": true,
+      "easyClientSwitching": true,
+      "platforms": [
+        "Linux",
+        "macOS",
+        "Windows"
+      ],
+      "ui": [
+        "CLI"
+      ],
+      "socials": {
+        "discord": "https://discord.com/invite/2BSdr2g",
+        "twitter": "https://twitter.com/stakewise_io",
+        "telegram": "https://t.me/stakewise_io",
+        "github": "https://github.com/stakewise"
+      },
+      "matomo": {
+        "eventCategory": "StakingProductCard",
+        "eventAction": "Clicked",
+        "eventName": "Clicked StakeWise go to link"
+      }
```

Comment:
Let's break these out into a separate PR

Reply:
Sure, I’ve excluded this part.